src/wallet/wallet.cpp

```diff
@@ -2819,7 +2820,11 @@ std::shared_ptr<CWallet> CWallet::Create(WalletContext& context, const std::stri
 
             LOCK(walletInstance->cs_wallet);
             if (walletInstance->IsWalletFlagSet(WALLET_FLAG_DESCRIPTORS)) {
-                walletInstance->SetupDescriptorScriptPubKeyMans();
+                SecureString mnemonic = gArgs.GetArg("-mnemonic", "").c_str();
+                SecureString mnemonic_passphrase = gArgs.GetArg("-mnemonicpassphrase", "").c_str();
+                gArgs.ForceRemoveArg("mnemonic");
```

Comment:
Please use `args` instead of `gArgs`, is available in `CWallet::Create()` scope

https://github.com/dashpay/dash/blob/41c64544ffcb1def67f3504b676af18fd732b0eb/src/wallet/wallet.cpp#L2815-L2818